tests/unit/components/plate_based_widgets/mapeditor/PlateMapEditor.spec.js

```diff
@@ -835,259 +307,29 @@ describe("PlateMapEditor.vue", () => {
-    // This condition affirms that the cross section of one_Column and A Row is matching its a proper way of developing test
-    // but due to jest rule of jest/no-conditional-expect this if condition is commented
-    // So had to comment the verification and just validate the expect part
-    //  if (plate_intersection_column === plate_intersection_row) {
-    expect(plate_intersection_row.attributes("stroke-width")).toBe("0");
-    // }
-    for (let i = 1; i < one_array.length; i++) {
-      const plate_well_column = wrapper.find(".plate_" + one_array[i]);
-      expect(plate_well_column.attributes("stroke-width")).toBe("0");
-    }
-    for (let j = 1; j < a_array.length; j++) {
-      const plate_well_row = wrapper.find(".plate_" + a_array[j]);
-      expect(plate_well_row.attributes("stroke-width")).toBe("4");
-    }
-    /* End of Scenario one */
-    /* Start Scenario Two
-     (a) Click             D Row
-     (b) Click+Shift      06 Column
-     (c) Click+Shift       D Row
-     Validate intersection Well 23 is not selected on verify D Row is deselected and remaining 06 Column plates are selected */
-
-    await D_row.trigger("click");
-    await six_column.trigger("click", {
-      shiftKey: true, // For testing @click.shift handlers
-    });
-    for (let i = 0; i < d_array.length; i++) {
-      const plate_well_column = wrapper.find(".plate_" + d_array[i]);
-      expect(plate_well_column.attributes("stroke-width")).toBe("4");
-    }
-    for (let j = 0; j < six_array.length; j++) {
-      const plate_well_row = wrapper.find(".plate_" + six_array[j]);
-      expect(plate_well_row.attributes("stroke-width")).toBe("4");
-    }
-    await D_row_negate.trigger("click", {
-      shiftKey: true, // For testing @click.shift handlers
-    });
-    plate_intersection_column = wrapper.find(
-      ".plate_" + d_array[d_array.length - 1]
-    );
-    plate_intersection_row = wrapper.find(
-      ".plate_" + six_array[six_array.length - 1]
-    );
-
-    // This condition affirms that the cross section of six Column and D Row is matching its a proper way of developing test
-    // but due to jest rule of jest/no-conditional-expect this if condition is commented
-    // So had to comment the verification and just validate the expect part
-    // if (plate_intersection_column === plate_intersection_row) {
-    expect(plate_intersection_row.attributes("stroke-width")).toBe("0");
-    expect(plate_intersection_column.attributes("stroke-width")).toBe("0");
-    // }
-    for (let i = 1; i < d_array.length; i++) {
-      const plate_well_column = wrapper.find(".plate_" + d_array[i]);
-      expect(plate_well_column.attributes("stroke-width")).toBe("0");
-    }
-    for (let j = 0; j < six_array.length - 1; j++) {
-      const plate_well_row = wrapper.find(".plate_" + six_array[j]);
-      expect(plate_well_row.attributes("stroke-width")).toBe("4");
+      await wrapper.vm.basic_shift_or_ctrl_select(well_indices[i]);
```

Comment:
similar question as above about why you switched from `trigger('click')` to `vm.method`

Reply:
Done.

Reply:
corrected to invoke using 'trigger('click')'